src/Component/Symbolizer/GraphicEditor/GraphicEditor.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import {
+  PointSymbolizer,
+  GraphicType,
+  MarkSymbolizer,
+  IconSymbolizer
+} from 'geostyler-style';
+import MarkEditor from '../MarkEditor/MarkEditor';
+import IconEditor, { DefaultIconEditorProps } from '../IconEditor/IconEditor';
+import GraphicTypeField, { DefaultGraphicTypeFieldProps } from '../Field/GraphicTypeField/GraphicTypeField';
+
+const _get = require('lodash/get');
+
+export interface DefaultGraphicEditorProps {
+  /** Label being used on TypeField */
+  graphicTypeFieldLabel: string;
+  /** Default GraphicTypeFieldProps */
+  graphicTypeFieldProps?: DefaultGraphicTypeFieldProps;
+  /** Default IconEditorProps */
+  iconEditorProps?: DefaultIconEditorProps;
+}
+
+// non default props
+interface GraphicEditorProps extends Partial<DefaultGraphicEditorProps> {
+  /** PointSymbolizer that is being used as graphic */
+  graphic: PointSymbolizer;
+  /** Currently selected GraphicType */
+  graphicType: GraphicType;
+  /** Gets called when changing a graphic */
+  onGraphicChange: ((graphic: PointSymbolizer) => void);
+}
+
+/** GraphicEditor to select between different graphic options */
+class GraphicEditor extends React.Component <GraphicEditorProps, {}> {
+
+  public static defaultProps: DefaultGraphicEditorProps = {
+    graphicTypeFieldLabel: 'Graphic-Type',
+    iconEditorProps: {
+      defaultIconSource: 'https://upload.wikimedia.org/wikipedia/commons/6/67/OpenLayers_logo.svg'
```

Comment:
I would also prefer a local dummy/fallback icon here.

Reply:
Added local icon and updated path in [this commit](https://github.com/terrestris/geostyler/pull/362/commits/b23d583e15ee6c6e73d1f37f47ba44a7a3126e88). Please note that I also changed the path to the icon in all other files, where a remote source was used.